README.md

```diff
@@ -144,8 +146,8 @@ Once you've built your image, you may want to run the tests:
 Assuming you've built `cockroachdb/cockroach`, let's run a simple Cockroach node in the background:
 
 ```bash
-$ docker run -p 8080:8080 -d cockroachdb/cockroach init \
-    -stores ssd="$(mktemp -d /tmp/dbXXX)"
+$ docker run -p 8080:8080 -d -v /cr-data cockroachdb/cockroach init /cr-data
```

Comment:
you don't want to daemonize the first one. Besides, don't you do an init-and-start in the next line? Seems like you don't need to run the first command and pass the volume if you do that.